content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -0,0 +1,170 @@
+- `task.external.execution.id`: The [external Task ID](https://docs.spring.io/spring-cloud-task/docs/2.2.0.M1/reference/#features-external_task_id) as present on the target platform (such as Cloud Foundry or Kubernetes) The type (Source, Processor, or Sink) of the application that reports the metrics
+- `task.parent.execution.id`: The [parent task ID](https://docs.spring.io/spring-cloud-task/docs/2.2.0.M1/reference/#features-parent_task_id) used to identify task that executes another task or tasks.
+
+If the Data Flow server is started with the `spring.cloud.dataflow.grafana-info.url` property pointing to your Grafana URL, the Grafana feature is enabled and Data Flow UI provides you with Grafana-buttons that can open a particular dashboard for a given stream, application, or application instance.
+
+As setting up InfluxDB is different depending on the platform on which you run, we provide instructions for each platform. In Spring Cloud Data Flow 2.x, local server and Cloud Foundry instructions for InfluxDB have been provided.
+
+## Local
+
+This section describes how to set up InfluxDB for a local machine.
+
+### InfluxDB
+
+InfluxDB is a popular open-source push-based time series database. It supports downsampling, automatically expiring and deleting unwanted data, and backup and restore. Analysis of data is done through an SQL-like query language.
+
+To enable Micrometer’s Influx meter registry for Spring Cloud Task application starters, start the Data Flow server with the following properties:
+
+```bash
+--spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.enabled=true
+--spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.db=myinfluxdb
+--spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.uri=http://localhost:8086
+--spring.cloud.dataflow.grafana-info.url=http://localhost:3000
+```
+
+Instead of having to install them manually, for a quick bootstrap, Spring Cloud Data Flow provides a [Docker Compose Influx](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-server/docker-compose-influxdb.yml) file which bring up Spring Cloud Data Flow, Skipper, Apache Kafka, Influx, and prebuilt dashboards for Grafana. Instructions below leverage this approach.
+
+[[tip | Upgrade to latest version of Docker ]]
+| We recommended that you upgrade to the [latest version](https://docs.docker.com/compose/install/) of Docker before running the `docker-compose` command. We have tested with Docker Engine version `18.09.2`.
+
+- Downloading the Docker Compose Influx file
+
+To download the Spring Cloud Data Flow Server Docker Compose file, run the following command:
+
+```bash
+wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/v%dataflow-version%/spring-cloud-dataflow-server/docker-compose-influxdb.yml
```

Comment:
Since `v%dataflow-version%` points to 2.1.2, and that doesn't include anything Task related, is this still correct?

If not, should we point to the YAML from 2.2.0.BS/master branch?

Reply:
Yeh, it seems that the only option for the moment is to hardcode the version to 2.2.0.M1. Also hardcode the:
```
export DATAFLOW_VERSION=2.2.0.M1
export SKIPPER_VERSION=2.1.0.M1
```

Reply:
But if you prefer the 2.2.0.BS it should do too?

Reply:
Yeah, BS should be cool.